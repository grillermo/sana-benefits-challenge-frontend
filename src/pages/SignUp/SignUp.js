import RegistrationForm from '../../components/RegistrationForm'

const SignUp = () => {
  return(
  <div>
    <h2>
      Sign up
    </h2>

    <RegistrationForm />
  </div>
  )
}

export default SignUp
