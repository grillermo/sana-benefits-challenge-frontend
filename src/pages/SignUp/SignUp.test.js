import { shallow } from 'enzyme';

import SignUp from './SignUp';

describe('SignUp', () => {
  test('renders successfully', () => {
    const tree = shallow(<SignUp />);

    expect(tree).toMatchSnapshot();
  });
});
