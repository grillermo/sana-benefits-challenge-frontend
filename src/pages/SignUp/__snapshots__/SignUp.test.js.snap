// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SignUp renders successfully 1`] = `
<div>
  <h2>
    Sign up
  </h2>
  <RegistrationFormContainer />
</div>
`;
